website/package.json

```diff
@@ -84,9 +84,10 @@
         "eslint-plugin-prefer-arrow": "^1.2.3",
         "eslint-plugin-react": "^7.35.2",
         "eslint-plugin-react-hooks": "^4.6.2",
+        "happy-dom": "^15.7.4",
         "jsdom": "^25.0.1",
```

Comment:
Could we get rid of the jsdom dependency then, if we decide to use happy-dom?

Reply:
I thought it might be used throughout in other places, but yeah, might be worth checking 👍 I'll do that.

Reply:
I was about to write that @fengelniederhammer, then I got interrupted 😄